### PR TITLE
[TT-9761] Add reload interval

### PR DESCRIPTION
**Pull request by zalbiraw**
Add ability to manage the reload interval to take the load off the gateway 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates go.mod in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/5418/merge
Commit:  
Triggered by: pull_request (@zalbiraw)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5859505584)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5418/merge
Commit: 0caf819354b469873f0c098d94f66f2a5ebf16df 
Triggered by: pull_request (@zalbiraw)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5859545657)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5418/merge
Commit: ba780fc88f34d018e75b093b9534c8321107529b 
Triggered by: pull_request (@titpetric)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6035706382)

**Review by jeffy-mathew — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5418/merge
Commit: 26fe9aedcfce79fa589a53089d6c7f09ba28605a 
Triggered by: pull_request (@titpetric)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6036782854)

File: config/config.go [hunk 1]
```diff
@@ -633,6 +633,10 @@ type Config struct {
 	// Disable dynamic API and Policy reloads, e.g. it will load new changes only on procecss start.
 	SuppressRedisSignalReload bool `json:"suppress_redis_signal_reload"`
 
+	// Sets the default interval for how often to reload the gateway when getting reload instructions in seconds.
```
Comment (dcs3spp) [inline config/config.go:636]: Sets the default wait interval in seconds to check for Gateway reload?

**Review by dcs3spp — CHANGES_REQUESTED** [review]
@titpetric

// Sets the default wait interval in seconds to check for Gateway reload?

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5418/merge
Commit: ebca49efd752b8655778d8ca415c533c3950ca68 
Triggered by: pull_request (@zalbiraw)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6037397380)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5418/merge
Commit: 317a5333021279c1704575fc7ee20017aa6b3297 
Triggered by: pull_request (@titpetric)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6037442723)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5418/merge
Commit: cf3aed71e4fcad9e3643ccc515b5b5664884d950 
Triggered by: pull_request (@titpetric)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/6037489671)

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**